baselines/webworker.generated.d.ts

```diff
@@ -641,18 +641,20 @@ declare var FileReaderSync: {
 };
 
 interface FormData {
-    append(name: string, value: string | Blob, fileName?: string): void;
+    append(name: string, value: string): void;
+    append(name: string, blobValue: Blob, filename?: string): void;
```

Comment:
the previous overload was better.. since you can pass a value of type `string | Blob`

Reply:
I agree, but somehow the spec authors have decided to use two signatures. No idea why, maybe documentation purpose?

```
[Constructor(optional HTMLFormElement form),
 Exposed=(Window,Worker)]
interface FormData {
  void append(USVString name, USVString value);
  void append(USVString name, Blob blobValue, optional USVString filename);
  void delete(USVString name);
  FormDataEntryValue? get(USVString name);
  sequence<FormDataEntryValue> getAll(USVString name);
  boolean has(USVString name);
  void set(USVString name, USVString value);
  void set(USVString name, Blob blobValue, optional USVString filename);
  iterable<USVString, FormDataEntryValue>;
};
```

Reply:
maybe we can override these two methods to avoid breaking existing users.

Reply:
Ah, I just noted this is a breaking change, but anyway `formData.append('str', 'str', 'str);` is invalid and throws on Firefox. I would say the spec signature is better to prevent unexpected runtime breakage.

Reply:
But the union problem... looks like there is no ideal solution here, I'll just override them to keep previous behavior.

Reply:
i agree.. but `append(string, sring|Blob)` should just work fine.